trade-finance-manager-ui/server/test-mocks/mock-banks.ts

```diff
@@ -13,7 +13,7 @@ const MONTHLY_REPORT_PERIOD_SCHEDULE = [
   { startMonth: 12, endMonth: 12 },
 ];
 
-const MOCK_BANKS = {
+export const MOCK_BANKS = {
```

Comment:
Seeing as you've added the `Bank` type below, could we add the `MOCK_BANKS: Bank[]` type assertion here?